### PR TITLE
Update Chart version

### PR DESCRIPTION
**Pull request by ke5C2Fin**
Update Chart version to 9.2.0

### CODE REVIEW

**Comment by ke5C2Fin** [comment]
Done, @Xtigyro 
Unless I should update the tests?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version `v0.2.4` is auto-generated.
 
+## [v9.2.1](https://github.com/puppetlabs/puppetserver-helm-chart/tree/v9.2.1) (2024-04-06)
```
Comment (Xtigyro) [inline CHANGELOG.md]: Not needed, please remove it.

File: Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: puppetserver
-version: 9.1.0
+version: 9.2.1
```
Comment (Xtigyro) [inline Chart.yaml]: Should be `9.2.0`.

**Review by Xtigyro — CHANGES_REQUESTED** [review]

**Comment by ldaneliukas** [comment]
@ke5C2Fin please do update the tests so that the CI checks aren't failing. You can do that by running `helm unittest . -u`.

**Review by Xtigyro — APPROVED** [review]

**Comment by Xtigyro** [comment]
@ke5C2Fin @ldaneliukas Thank you both! 🥇

**Pull request merged**